test/extended/testdata/build-secrets/s2i-binary-dir/Dockerfile

```diff
@@ -0,0 +1,9 @@
+USER default
+EXPOSE 8080
+ENV RACK_ENV production
+ENV RAILS_ENV production
+COPY . /opt/app-root/src/
+RUN scl enable rh-ruby22 "bundle install"
+CMD ["scl", "enable", "rh-ruby22", "./run.sh"]
+
+USER default
```

Comment:
what is this docker file used for?  s2i builds don't normally need dockerfiles, so it's a bit confusing to have it here.